modules/toolbox/framework_tools.py

```diff
@@ -1,8 +1,11 @@
 from modules.toolbox import *
+import pandas as pd
+from sklearn import preprocessing
+from sklearn import cross_validation
 
 # Helper functions for the whole framework to use
 def get_train_test(pandas_data, target_col):
-    pandas_data = scale_numeric_data(pandas_data)
+    #pandas_data = scale_numeric_data(pandas_data)
```

Comment:
Just wondering why this is commented out. Any particular reason? Scaling is important for removing bias towards one variable or another